src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -80,6 +80,11 @@ public static void checkUserIsAdmin(Request req, Response res) {
         }
     }
 
+    /** Check if the incoming user is an admin user */
+    public static boolean isUserAdmin(Auth0UserProfile user) {
```

Comment:
Make method `checkUserIsAdmin` call `isUserAdmin`.

Reply:
Actually, I wouldn't worry about making this change. I've added @br648's `isUserAdmin` method to #14 and am using the method in `checkUserIsAdmin` in that downstream PR. So I'd rather if we just leave that unchanged to avoid potential (even if unlikely) conflicts.